src/sdk/model/edge/firewall/deprecated/edge-firewall-rule.ts

```diff
@@ -0,0 +1,155 @@
+import { EdgeFirewallRuleJson } from './__json__/edge-firewall-rule-json';
+
+/**
+ * FirewallRule class
+ */
+export class EdgeFirewallRule {
```

Comment:
did you generate these with the IDEA plugin or write them manually?

Reply:
This class is the old one ( everything that is in /deprecated/folder is old classes ). I'm not sure this one was generated with the plugin i just moved and eventually renamed those old classes.

All new classes are generated with IDEA plugin.